python/cudf/cudf/core/index.py

```diff
@@ -542,7 +542,7 @@ def as_column(self):
         )
 
     def to_gpu_array(self):
-        return self.as_column().to_gpu_array()
+        return self.as_column().data.mem
```

Comment:
Should keep this one as `to_gpu_array()` as we may want it to return by copy in the future and it needs to handle nulls.

Reply:
Awesome. Thanks!

Reply:
Shouldn't change this one per previous conversation.